webServer.js

```diff
@@ -308,6 +319,13 @@ function start() {
     }
     var licenses = librarylicensesList(library);
     var assets = libraryAssetsList(library, version);
+    var stargazers_count = null, forks = null, subscribers_count = null;
+    var metaInfo = libraryMetaInfo(library);
+    if (metaInfo !== null && metaInfo !== undefined) {
```

Comment:
When the library with GitHub url but we cannot access it, it'll be `undefined`. 
For example: `radiobox.css` with GitHub url: https://github.com/720kb/radiobox but 404 not found. We don't have popularity information for this kind of libraries.